OpenRA.Mods.Common/Traits/Armament.cs

```diff
@@ -65,6 +65,9 @@ public class ArmamentInfo : PausableConditionalTraitInfo, Requires<AttackBaseInf
 		[Desc("Condition to grant while reloading.")]
 		public readonly string ReloadingCondition = null;
 
+		[Desc("Whether the weapon is assumed to be reloaded if enough time has passed for a reload.")]
+		public readonly bool AutoReload = true;
```

Comment:
Perhaps something like `ReloadWhileWaiting` would be more descriptive (and not confused with self reloading). Opinions?